app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -228,8 +231,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 UiSettings uiSettings = mapBoxMap.getUiSettings();
                 uiSettings.setCompassGravity(Gravity.BOTTOM | Gravity.LEFT);
                 uiSettings.setCompassMargins(12, 0, 0, 24);
-                uiSettings.setLogoEnabled(false);
-                uiSettings.setAttributionEnabled(false);
+                uiSettings.setLogoEnabled(true);
```

Comment:
Is this related with this PR?

Reply:
No. I guess I mixed these up while updating my fork. These have been reverted.